### ISSUE TITLE
Improper to assume current value of properties to be the same as their default value

### ISSUE THREAD

**Issue by ywchan2005** [Motion-Project/motion#1549]
0. Reviewed guide and contributing documents? (Yes/No): yup
1. version [x.y.z, hash, other]: 4.4.0
2. installed as a package or compiled from sources [deb, rpm, git, other]: compiled from git source
3. standalone or part of third party [motion, MotionEyeOS, other]: standalone
4. video stream source [V4L (card or USB), net cam (mjpeg, rtsp, other), mmal]: v4l2
5. hardware [x86, ARM, other]: x86
6. operating system [16.04, Stretch, etc, FreeBSD, other]: ubuntu 20.04

### Cause

While motion starts, it asks the camera for the default values of each configurable property and uses these values as their current values.
```
curdev->devctrl_array[indx_ctrl].ctrl_currval = vid_ctrl.default_value;
```
However, the camera may have values which are previously configured stored inside.

### Problem

Motion is asked to set a value for one of the property, say setting brightness to 50, according to motion.conf.
```
video_params brightness=50
```
Unfortunately the default value and the stored value are 50 and 0 respectively. As motion checks to see whether the current value (`ctrl_currval`), which is the default value 50, is different from the value being set (`ctrl_newval`), which is 50 according to motion.conf, before actually running the function to set the value,
```
if (devitem->ctrl_currval != devitem->ctrl_newval) {
    memset(&vid_ctrl, 0, sizeof (struct v4l2_control));
    vid_ctrl.id = devitem->ctrl_id;
    vid_ctrl.value = devitem->ctrl_newval;
    retcd = xioctl(vid_source, VIDIOC_S_CTRL, &vid_ctrl);
}
```
it fails to do so.

**Comment by Mr-Dave** [comment]
Looking at the properties of the struct I notice that there isn't any "current value".  As such I'm thinking the solution is to wipe out any previously set values for the device and set them to the defaults.

**Comment by ywchan2005** [comment]
the current value can be retrieved by `xioctl(vid_source, VIDIOC_G_CTRL, &control)`

**Comment by Mr-Dave** [comment]
Thanks for reporting.  Closed via 94a2ef1367f196a1401fa804d5c241ca6ff4f5b7
